src/User.java

```diff
@@ -8,11 +8,11 @@
 package reading_bubby.src;
 
 import java.io.*;
-import java.time.LocalDate;
+//import java.time.LocalDate;
```

Comment:
this change and those on lines 14, 15, 66, and 88 are a result of moving a function from this file to ReadingBuddy.java (because the main ended up needing to have all the group posts so it could remove the appropriate ones when a group is ended)